core/db/default/spree/countries.rb

```diff
@@ -1,14 +1,19 @@
 require 'carmen'
 
-Carmen::Country.all.each do |country|
-  Spree::Country.where(
-    name: country.name,
-    iso3: country.alpha_3_code,
-    iso: country.alpha_2_code,
-    iso_name: country.name.upcase,
-    numcode: country.numeric_code,
-    states_required: country.subregions?
-  ).first_or_create
+if Spree::Country.all.blant?
```

Comment:
@r3cha this should be `blank` I suppose 😂

Reply:
Yed tight %)